scripts/clangfmt

```diff
@@ -12,9 +12,8 @@
 set -euo pipefail
 IFS=$'\n\t'
 
-# The required version of clang-format.
-CLANG_FORMAT_VERSION=5.0
-CLANG_FORMAT_VERSION_STRING="clang-format version $CLANG_FORMAT_VERSION"
+# The required version of clang-format (matches minimum clang version) for CI
```

Comment:
Header of this file contains the comment `clang-format is from the most recent version of LLVM supported by Scala Native`, but we're using the opposite. We should fix it as it might be misleading in the future.

Reply:
Thanks for the review. I have fixed up the comment. I was really surprised too that it would be so inconsistent between versions. I was hopeful we wouldn't need special setup and was surprised that older versions don't have the `--dry-run` option.